### PR TITLE
[License] Add license

### PR DESCRIPTION
**Pull request by EmmyMiao87**
#4370
Change-Id: I5e30a29e8150c6a1a71878822c39960f0cf31e18

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [] Bugfix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation Update (if none of the other choices apply)
- [] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [] I have create an issue on (Fix #ISSUE), and have described the bug/feature there in detail
- [] Compiling and unit tests pass locally with my changes
- [] I have added tests that prove my fix is effective or that my feature works
- [] If this change need a document change, I have updated the document
- [] Any dependent changes have been merged

### CODE REVIEW

File: be/test/plugin/plugin_test/source/test.zip.md5 [hunk 1]
```diff
@@ -1 +1,20 @@
+<!--
```
Comment (morningman) [inline be/test/plugin/plugin_test/source/test.zip.md5:1]: Do not add license for this file

File: fe/fe-core/src/test/resources/data/net_snmp_normal [hunk 2]
```diff
@@ -1,3 +1,20 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```
Comment (morningman) [inline fe/fe-core/src/test/resources/data/net_snmp_normal:1]: Do not add license for this file

File: fe/fe-core/src/test/resources/plugin_test/source/test.zip.md5 [hunk 3]
```diff
@@ -1 +1,18 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```
Comment (morningman) [inline fe/fe-core/src/test/resources/plugin_test/source/test.zip.md5:1]: Do not add license for this file

File: fe/fe-core/src/test/resources/spark_launcher_monitor.log [hunk 4]
```diff
@@ -1,3 +1,20 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```
Comment (morningman) [inline fe/fe-core/src/test/resources/spark_launcher_monitor.log:1]: Do not add license for this file

File: be/test/plugin/plugin_test/source/test.zip.md5 (hunk 1 above)
Comment (EmmyMiao87) [reply be/test/plugin/plugin_test/source/test.zip.md5:1]: Should I need to add documents to the exemption list?

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**